core/common/src/TextureMapping.ts

```diff
@@ -9,18 +9,38 @@
 import { IndexedPolyfaceVisitor, Matrix3d, Point2d, Point3d, PolyfaceVisitor, Transform, Vector3d } from "@itwin/core-geometry";
 import { RenderTexture } from "./RenderTexture";
 
+/** Defines normal map parameters and texture to use as pattern map when texture is a normal map.
+ * @public
+ */
+export class NormalMap {
```

Comment:
`NormalMap` will be easier to use and extend if it's an interface instead of a class.

Reply:
done